tests/bluetooth/bsim_bt/bsim_test_advx/src/main.c

```diff
@@ -545,6 +545,38 @@ static void test_advx_main(void)
 	}
 	printk("success.\n");
 
+	uint8_t num_adv_sets;
+	num_adv_sets = ll_adv_aux_set_count_get();
+
+	printk("Creating %d adv sets ...", num_adv_sets);
+	for (handle = 0; handle < num_adv_sets; ++handle) {
+		err = ll_adv_params_set(handle, evt_prop, ADV_INTERVAL, adv_type,
+					OWN_ADDR_TYPE, PEER_ADDR_TYPE, PEER_ADDR,
+					ADV_CHAN_MAP, FILTER_POLICY, ADV_TX_PWR,
+					phy_p, ADV_SEC_SKIP, phy_s, ADV_SID,
+					SCAN_REQ_NOT);
+		if (err) {
+			goto exit;
+		}
+	}
+	printk("success.\n");
+
+	printk("Clearing all adv sets...");
+	err = ll_adv_aux_set_clear();
+	if (err) {
+		goto exit;
+	}
+	printk("success.\n");
+
+	printk("Trying to remove adv sets ...");
+	for (handle = 0; handle < num_adv_sets; ++handle) {
+		err = ll_adv_aux_set_remove(handle);
+		if (err != BT_HCI_ERR_UNKNOWN_ADV_IDENTIFIER) {
+			goto exit;
+		}
+	}
+	printk("success.\n");
```

Comment:
We are missing a test for the case that you try to clear all sets while one or more are still enabled

Reply:
Changed test to enable every other set, that should cover the case of only some enabled.

Reply:
Still missing the case where the function returns BT_HCI_ERR_CMD_DISALLOWED

Reply:
Add a test for that.

Reply:
Added a test for that.